LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2013-2018 tty-clock contributors
+Copyright (c) 2008-2009 Martin Duquesnoy <xorg62@gmail.com>
```

Comment:
maybe i'm nitpicking, but i would keep the copyright attribution as it is in the ttyclock.c file, unless it is also changed there as well.

Reply:
Not nipicking, just my PR wasn't complete.  I have added another commit for the source files, also fixed the year in `LICENSE`, somehow I misread the correct starting year of contributions while skimming through commits.  I preset the end year as 2018 in the license and main C code as I believe the main code would get contributions, the year is still young.